docs/doxygen/Doxyfile

```diff
@@ -943,8 +943,8 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ../../README.md \
-                        ../../amd_openvx/openvx/include/VX/vx.h \
+#INPUT                  = ../README.md \
```

Comment:
Why did you comment out the old stuff in this file (rather than delete it)?

Reply:
In case we wanted to restore it.